xls2xml/tests/test_TSVReader.py

```diff
@@ -8,16 +30,16 @@ def test_is_valid():
     tsv_reader = TSVReader('data/example_samples.tsv', 'data/T2D_xls2xml_v1.conf', 'Sample')
     assert tsv_reader.is_valid()
 
-def test_get_headers():
+def test_get_current_headers():
     tsv_reader = TSVReader('data/example_samples.tsv', 'data/T2D_xls2xml_v1.conf', 'Sample')
-    headers = tsv_reader.get_headers()
+    headers = tsv_reader.get_current_headers()
     assert isinstance(headers, list)
     assert set(headers) == {'Sample_ID', 'Subject_ID', 'Geno_ID', 'Phenotype', 'Gender', 'Analysis_alias', 'Cohort ID',
                             'Ethnicity', 'Ethnicity Description', 'T2D', 'Case_Control', 'Description', 'Center_name',
                             'Hispanic or Latino; of Spanish origin', 'Age', 'Year of Birth', 'Year of first visit',
                             'Cell Type', 'Maternal_id', 'Paternal_id', 'Novel Attributes'}
 
-def test_next_row():
+def test_next():
```

Comment:
can we please include test for empty row at the end of the tsv file

Reply:
I have added an empty line at the end of the example_samples.tsv file and added iteration in test_next() to reach the end of the file without seeing any issues. Fixed in 27f748bc